components/more_direct_channels/__snapshots__/more_direct_channels.test.jsx.snap

```diff
@@ -189,7 +189,9 @@ exports[`components/MoreDirectChannels should match renderOption snapshot 1`] =
     <div
       className="more-modal__name"
     >
-      <span>
+      <span
+        id="displayedUserNameundefined"
```

Comment:
Could you please update [related unit test for snapshot](https://github.com/mattermost/mattermost-webapp/blob/master/components/more_direct_channels/more_direct_channels.test.jsx#L175) so it will not reflect as undefined?  Thanks

```diff
- expect(wrapper.instance().renderOption({id: 'user_id_1', delete_at: 0}, true, jest.fn())).toMatchSnapshot();
+ expect(wrapper.instance().renderOption({id: 'user_id_1', username: 'username1', delete_at: 0}, true, jest.fn())).toMatchSnapshot();
```

Reply:
Good catch ! Thanks ! I'll be mindful of these types of issues in the future.